scripts/profiles/django-simple/run.sh

```diff
@@ -5,7 +5,7 @@ set -eu
 PREFIX=${1}
 
 AUSTIN_INTERVAL=200  # usec
-AUSTIN_EXPOSURE=5  # sec
+AUSTIN_EXPOSURE=3  # sec
```

Comment:
seems unrelated?

Reply:
I was noticing timeouts, which seemed to be caused by the resolve step taking too long. I'll try to revert this and see if that was not just a fluke. Otherwise, if I change this again it's because of this reason 🙁